Main.java

```diff
@@ -5,61 +5,58 @@
 
 public class Main {
 
-    public static void main(String[] args){
+    public static void main(String[] args) {
 
-        if(args.length!=2)
-        {
+        if (args.length != 2) {
             System.out.println("File path and search word cannot be empty");
-            System.exit(0);
+            return;
         }
-        String path=args[0];
-        String search=args[1];
-        File file=new File(path);
+        String path = args[0];
+        String search = args[1];

```

Comment:
Can you rename this to a more understandable variable name?

Reply:
Are you referring to path and search?

Reply:
I have renamed them to textFilePath and searchKeyword respectively.